dev/src/dev/liquibase.clj

```diff
@@ -0,0 +1,32 @@
+(ns dev.liquibase
+  (:require [clojure.string :as str]
+            [colorize.core :as colorize]
+            [metabase.db.data-source :as mdb.data-source]
+            [metabase.db.env :as mdb.env]))
+
+(comment mdb.data-source/keep-me)
+
+(defn -main
+  "Use the Liquibase CLI with `clojure -M:liquibase <command>`."
+  [& args]
+  (let [args (if (empty? args)
+               ["help"]
+               args)
+        args (into ["--changeLogFile=resources/migrations/000_migrations.yaml"]
+                   (comp cat
+                         (filter seq))
+                   (let [^metabase.db.data_source.DataSource data-source mdb.env/data-source
+                         ^java.util.Properties properties                (.properties data-source)]
+                     [(when-let [user (some-> properties (.get "user"))]
+                        ["--username" user])
+                      (when-let [password (some-> properties (.get "password"))]
```

Comment:
I get an error `Could not parse '--password'` when I run the liquibase command, since `password` exists in `data-source` for me but is an empty string. This would fix that.
```suggestion
                      (when-let [password (seq (some-> properties (.get "password")))]
```